modules/persistent-storage-local-install.adoc

```diff
@@ -19,6 +19,19 @@ The Local Storage Operator is not installed in {product-title} by default. Use t
 $ oc new-project local-storage
 ----
 
+. Optional: Add an annotation to the project that blocks a default node selector.
++
+You might want to use the Local Storage Operator to create volumes on infrastructure nodes in support of components such as logging and monitoring.
++
+However, clusters with infrastructure nodes are likely to use a default node selector that includes only worker nodes. To comply with licensing, this prevents the scheduler from running user apps on infrastructure nodes.
```

Comment:
I think this needs one additional sentence at the end to fully indicate that the LSO wouldn't be able to be scheduled on these nodes with the default permissions. Perhaps it's overkill, but maybe something similar to the following added at the end of this paragraph:

> Without adjusting the default node selector, the Local Storage Operator will only be scheduled on worker nodes.

Reply:
Yes, I agree it's a good idea to be more explicit. I slightly worded your suggested sentence: 
> You must adjust the default node selector so that the Local Storage Operator includes the infrastructure nodes, and not just worker nodes.